components/molecules/SearchBar/index.tsx

```diff
@@ -1,14 +1,15 @@
-import { useCallback, useRef, useState } from "react";
+"use client";
+import { useCallback, useRef, useState, useId } from "react";
 import classNames from "classnames";
 import { useTranslation } from "react-i18next";
 import { useOnClickOutside, useKeyDownEvent } from "@/hooks/listeners";
 import { IconComposer } from "@rubin-epo/epo-react-lib";
 import useQueryParams from "@/lib/routing/useQueryParams";
 import { useRouter } from "next/navigation";
+import styles from "./styles.module.scss";
 
-const INPUT_ID = "headerSearchBar";
-
-export default function SearchBar() {
+const SearchBar: FunctionComponent = () => {
+  const id = useId();
```

Comment:
since this component is rendered multiple times but display is controlled by media queries, use a randomly generated ID instead of static to avoid element ID collisions